internal/pkg/workspace/workspace.go

```diff
@@ -151,6 +153,52 @@ func (ws *Workspace) ListWorkloads() ([]string, error) {
 	})
 }
 
+// ListPipelines returns the names of all pipelines in the workspace.
+func (ws *Workspace) ListPipelines() (map[string]string, error) {
+	pipelines := make(map[string]string)
+	// Look for legacy pipeline.
+	legacyPath, err := ws.PipelineManifestLegacyPath()
+	if err != nil {
+		return nil, err
+	}
+	manifest, err := ws.ReadPipelineManifest(legacyPath)
+	if err != nil {
+		return nil, err
+	}
+	pipelines[manifest.Name] = legacyPath
```

Comment:
Shouldn't we skip if we receive `ErrNoPipelineInWorkspace` instead of returning the error?